pkg/ddl/job_submitter.go

```diff
@@ -637,33 +674,21 @@ func buildJobDependence(t *meta.Meta, curJob *model.Job) error {
 	return nil
 }
 
-func (e *executor) notifyNewJobSubmitted(ch chan struct{}, etcdPath string, jobID int64, jobType string) {
-	// If the workers don't run, we needn't notify workers.
-	// TODO: It does not affect informing the backfill worker.
-	if !config.GetGlobalConfig().Instance.TiDBEnableDDL.Load() {
```

Comment:
if `!config.GetGlobalConfig().Instance.TiDBEnableDDL.Load()` is true, then it's not the owner, we need notify by etcd